src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalService.kt

```diff
@@ -87,15 +96,23 @@ class CalculationTransactionalService(
     calculationType: CalculationStatus = PRELIMINARY,
   ):
     ValidationResult {
-    var messages = validationService.validateBeforeCalculation(providedSourceData, calculationUserInputs) // Validation stage 1 of 3
+    val bulkCalculationReason = calculationReasonRepository.findTopByIsBulkTrue()
```

Comment:
I think we should probably move this lookup into the Bulk Calculation Service, and pass in an argument to this method incase its reused elsewhere.